### ISSUE TITLE
 1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-video#38]
Wenn ein Widget-Titel leer ist, wird es dennoch gesetzt.
Dadurch wird dann die WCAG  1.3.1 Info and Relationships  "Heading is missing text"
 ausgelöst

**Comment by xwolfde** [comment]
Weitere Meldung für den PLay-Button:

Warning: "i" tag used to format text

Einfach auf em wechseln :)

**Comment by iamwebrocker** [comment]
Kannst Du ein Beispiel Link geben? Im widget template wird geprüft, ob showtitle da ist, und nur dann heading h mit title zurück gegeben. Funkt uns da evtl ein WordPress default widget verhalten dazwischen?

**Comment by iamwebrocker** [comment]
kann es sein, dass ds aus dem theme kommt? da werden für widgets allgemein die title tags gesetzt:

https://github.com/RRZE-Webteam/FAU-Einrichtungen/blob/60f7840282b0e2e7604d768efbbe6191434506d9/functions/widgets.php#L24

**Comment by xwolfde** [comment]
Ich schau noch mal. Dei Fehler kommen von Beta....

äh... ah. Moment, ok, vergisst es, dass ist die ANtwort. da ist die alte version /
